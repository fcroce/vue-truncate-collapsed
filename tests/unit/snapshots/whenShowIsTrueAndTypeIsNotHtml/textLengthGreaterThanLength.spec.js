import Vue from "vue";
import Truncate from "../../../../src/truncate.vue";

const component = Vue.extend(Truncate);
const vm = new component({
  propsData: {
    text: "MOCKED",
    length: 1
  },
  data: {
    show: true
  }
}).$mount();

describe("Test truncate component snapshots", () => {
  it("should match the snapshot for text.length > length", () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
