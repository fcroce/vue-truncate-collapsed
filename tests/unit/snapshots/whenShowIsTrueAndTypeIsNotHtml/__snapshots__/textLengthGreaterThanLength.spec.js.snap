// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test truncate component snapshots should match the snapshot for text.length > length 1`] = `
<div>
  <!---->
   
  <div>
    <span>
      MOCKED
    </span>
     
    <a
      class=""
    >
      Show Less
    </a>
  </div>
</div>
`;
